mubench.reviewsite/setup/db_table_utils.php

```diff
@@ -0,0 +1,159 @@
+<?php
+use Illuminate\Database\Schema\Blueprint;
+
+require_once __DIR__ . "/TableCreationHelper.php";
+
+function createTables($connection)
```

Comment:
I like that this is a function now that we can call with the connection. I'm wondering whether we want to go one step further an make this a method on the `TableCreationHelper` (which should then be called something like `SchemaSetup`). Then we could instantiate a setup for a connection and just `run()` it. What do you think?